doc/source/data/working-with-llms.rst

```diff
@@ -88,6 +88,25 @@ Some models may require a Hugging Face token to be specified. You can specify th
         batch_size=64,
     )
 
+If your model is hosted on AWS S3, you can specify the S3 path in the `model_source` argument, and specify `load_format="runai_streamer"` in the `engine_kwargs` argument.
+
+.. note::
+    Install vLLM with runai dependencies: `pip install -U "vllm[runai]==0.7.2"`
+
+.. testcode::
+
+    config = vLLMEngineProcessorConfig(
+        model_source="s3://your-bucket/your-model/",  # Make sure adding the trailing slash!
+        engine_kwargs={"load_format": "runai_streamer"},
+        runtime_env={"env_vars": {
+            "AWS_ACCESS_KEY_ID": "your_access_key_id",
+            "AWS_SECRET_ACCESS_KEY": "your_secret_access_key",
+            "AWS_REGION": "your_region",
+        }},
+        concurrency=1,
+        batch_size=64,
+    )
```

Comment:
Can you move this down to this section / replace the below?

https://github.com/ray-project/ray/pull/51103/files#diff-3535599cd6d3c98d0a0c0b28c5274136a85d83bac96cd4b3349277856926dd68L130